plugin/pxr/maya/lib/pxrUsdMayaGL/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Unit test scripts.
+set(TEST_SCRIPT_FILES
+    # Note: the following test fails when switching to the "Expanded"
+    #       assembly mode, so disabled it until Pixar figures
+    #       what is going wrong.
```

Comment:
Is this comment still valid? If you remove it don't bother re-running the preflight. Just let me know and I'll merge.

Reply:
Good catch, yeah, that comment is obsolete

Reply:
Oh sorry I missed this - we fixed this with https://github.com/Autodesk/maya-usd/pull/3492 so you should be able to re-enable this test.

Reply:
Removed the comment, can commit